demos/ble/CMakeLists.txt

```diff
@@ -20,35 +20,13 @@ afr_module_dependencies(
         AFR::ble
 )
 
+if( EXISTS ${CMAKE_CURRENT_LIST_DIR}/mqtt_ble )
```

Comment:
For the separate CMakeLists.txt to be correctly added to the metadata, it needs to be included with `afr_add_subdirectory`. 
@tgsong , can you confirm that?

Reply:
Yes. And for *.cmake file, it can be part of library source files as a workaround.

Reply:
Done.